src/_rules/color.js

```diff
@@ -1,3 +1,15 @@
 export const opacity = [
   [/^opacity-?(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
 ];
+
+export const textColors = [
+  ['text', { color: 'var(--w-text-color)' }],
+  ['text-inverted', { color: 'var(--w-text-color-inverted)' }, { autocomplete: 'text-inverted' }],
```

Comment:
autocomplete isn't needed for 'static' rules (only ones with regex)

Reply:
Fixed